src/api/Nhs.Appointments.Persistance/BookingCosmosDocumentStore.cs

```diff
@@ -10,21 +10,23 @@ public class BookingCosmosDocumentStore : IBookingsDocumentStore
     private const int PointReadLimit = 3;
     private readonly ITypedDocumentCosmosStore<BookingDocument> _bookingStore;
     private readonly ITypedDocumentCosmosStore<BookingIndexDocument> _indexStore;
+    private readonly TimeProvider _time;
 
-    public BookingCosmosDocumentStore(ITypedDocumentCosmosStore<BookingDocument> bookingStore, ITypedDocumentCosmosStore<BookingIndexDocument> indexStore) 
-    { 
+    public BookingCosmosDocumentStore(ITypedDocumentCosmosStore<BookingDocument> bookingStore, ITypedDocumentCosmosStore<BookingIndexDocument> indexStore, TimeProvider time)
+    {
         _bookingStore = bookingStore;
         _indexStore = indexStore;
+        _time = time;
     }
-           
+
     public Task<IEnumerable<Booking>> GetInDateRangeAsync(DateTime from, DateTime to, string site)
     {
         return _bookingStore.RunQueryAsync<Booking>(b => b.Site == site && b.From >= from && b.From <= to);
     }
 
-    public async Task<IEnumerable<Booking>> GetCrossSiteAsync(DateTime from, DateTime to)
+    public async Task<IEnumerable<Booking>> GetCrossSiteAsync(DateTime from, DateTime to, bool provisional = false)
     {
-        var bookingIndexDocuments = await _indexStore.RunQueryAsync<BookingIndexDocument>(i => i.From >= from && i.From <= to);
+        var bookingIndexDocuments = await _indexStore.RunQueryAsync<BookingIndexDocument>(i => i.From >= from && i.From <= to && i.Provisional == provisional);
```

Comment:
We might want to look at this query and decide if it is better to do this on cosmos or in code. If we only expect a small number of provisionals to come in we might actual find that the RUs are lower if we don't have this on the query. Worth checking by trying it in the cosmos emu